compliance/initialize_duvet.sh

```diff
@@ -0,0 +1,7 @@
+#/usr/bin/env bash
+
+duvet extract https://tools.ietf.org/rfc/rfc5869 # HMAC-based Extract-and-Expand Key Derivation Function (HKDF)
```

Comment:
One thing to note is `tools.ietf.org` now redirects to `www.rfc-editor.org`. It doesn't have to be a part of this PR but it would be good to use the canonical URL instead (see https://github.com/aws/s2n-quic/pull/1147).

Reply:
Okay, that makes sense. I think the easiest way to do this might be to wait until all the HRR PRs are merged and then fix this in a single PR, if that sounds reasonable. I created and assigned myself to #3377.